examples/src/read-data.c

```diff
@@ -0,0 +1,241 @@
+/*
+ * Copyright (c) 2020, Lawrence Livermore National Security, LLC.
+ * Produced at the Lawrence Livermore National Laboratory.
+ *
+ * Copyright 2019, UT-Battelle, LLC.
+ *
+ * LLNL-CODE-741539
+ * All rights reserved.
+ *
+ * This is the license for UnifyFS.
+ * For details, see https://github.com/LLNL/UnifyFS.
+ * Please read https://github.com/LLNL/UnifyFS/LICENSE for full license text.
+ */
+#include <config.h>
+
+#include <stdio.h>
+#include <stdlib.h>
+#include <stdint.h>
+#include <string.h>
+#include <errno.h>
+#include <limits.h>
+#include <sys/time.h>
+#include <sys/types.h>
+#include <unistd.h>
+#include <fcntl.h>
+#include <libgen.h>
+#include <getopt.h>
+#include <unifyfs.h>
+
+#include "testutil.h"
+
+static int unifyfs;
+static char filename[PATH_MAX];
+static char* mountpoint = "/unifyfs";
+
+static char linebuf[LINE_MAX];
+static char* buf;
+static unsigned int bufsize;
+
+static int fd;
+static struct stat sb;
+static unsigned int offset;
+static unsigned int length;
```

Comment:
These should be `unsigned long` to match the `%lu` identifiers in `printf(" -> pread(off=%lu, len=%lu) = %d"`